src/utils/getFontsPath.js

```diff
@@ -0,0 +1,5 @@
+import path from 'path';
+
+export default (bootstrapRelPath) => {
+  return path.join(bootstrapRelPath, 'assets', 'fonts', 'bootstrap/');
```

Comment:
I thought `path` takes care of it, no? I haven't tested it on Windows though.

https://nodejs.org/api/path.html#path_path_normalize_p